specification/eventhub/resource-manager/Microsoft.EventHub/preview/2023-01-01-preview/Clusters-preview.json

```diff
@@ -0,0 +1,493 @@
+          "202": {
+            "description": "Event Hubs Cluster delete request accepted."
+          },
+          "204": {
+            "description": "No content."
+          },
+          "default": {
+            "description": "Event Hubs error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters/{clusterName}/namespaces": {
+      "get": {
+        "tags": [
+          "EventHubs Clusters Namespace"
+        ],
+        "operationId": "Clusters_ListNamespaces",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "description": "List all Event Hubs Namespace IDs in an Event Hubs Dedicated Cluster.",
+        "x-ms-examples": {
+          "ListNamespacesInCluster": {
+            "$ref": "./examples/Clusters/ListNamespacesInClusterGet.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "List of Namespace IDs in Cluster successfully delivered.",
+            "schema": {
+              "$ref": "#/definitions/EHNamespaceIdListResult"
+            }
+          },
+          "default": {
+            "description": "Event Hubs Cluster error response detailing why this operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Cluster": {
+      "type": "object",
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/ClusterSku",
+          "description": "Properties of the cluster SKU."
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system meta data relating to this resource.",
+          "$ref": "../../../common/v1/definitions.json#/definitions/systemData"
+        },
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "createdAt": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The UTC time when the Event Hubs Cluster was created."
+            },
+            "provisioningState": {
+              "readOnly": true,
+              "type": "string",
```

Comment:
shouldn't this be an enum?

Reply:
We have seen many a serialization issues in our SDK's due to missing enum values in state. If allowed, we would like to keep this string, if not I can gather all states and add them here.